rrmngmnt/host.py

```diff
@@ -21,6 +21,7 @@
 from rrmngmnt.resource import Resource
 from rrmngmnt.service import Systemd, SysVinit, InitCtl
 from rrmngmnt.storage import NFSService, LVMService
+from utilities import utils
```

Comment:
this module is not listed as dependency, and I am afraid it is not publicly available anyway